sandbox/docker/master/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.9-slim AS dependencies
+
+WORKDIR /tmp
+COPY ./requirements.txt /tmp/
+
+RUN apt-get update \
+        && apt-get install -y --no-install-recommends git \
+        && pip install -r requirements.txt \
```

Comment:
We need to remove `\` at the end of the line. Could you try building the Docker image by yourself before pushing the branch next time?

Reply:
Yeah sorry, my machine was being funky and docker wasn't working.